static/templates/user_info_popover_content.handlebars

```diff
@@ -6,6 +6,9 @@
             {{#if is_active }}
                 {{#if is_bot}}
                 <i class="zulip-icon bot" aria-hidden="true"></i>
+                {{else if is_admin}}
+                <i class="zulip-icon crown" aria-hidden="true"></i>
+                <span class="user-status-indicator popover_user_presence" title="{{user_last_seen_time_status}}"></span>
```

Comment:
Don't copy code for this; just move your if statement inside the `else` clause below.

Reply:
Updated the pull request with this change.